tests/behat/features/create-taxonomies.feature

```diff
@@ -40,4 +40,4 @@ Feature: Create taxonomies
     And I click the "Children" CMS tab
     Then I should see "My child taxonomy term"
```

Comment:
```suggestion
    Then I should see "My child taxonomy term" in the "#Form_EditForm_SilverStripe-Taxonomy-TaxonomyTerm" element
```
If we're being specific below, we need to be specific here too.

Reply:
Updated